public/plugin.ts

```diff
@@ -112,7 +112,14 @@ export class SecurityPlugin
           excludeFromDisabledTransportCategories(config.disabledTransportCategories.exclude);
           excludeFromDisabledRestCategories(config.disabledRestCategories.exclude);
 
-          return renderApp(coreStart, depsStart as SecurityPluginStartDependencies, params, config);
+          return renderApp(
+            coreStart,
+            depsStart as SecurityPluginStartDependencies,
+            params,
+            config,
+            params.setHeaderActionMenu,
```

Comment:
we are already passing params object here. Re-use it?

Reply:
Sure, good catch